src/main/java/dev/openfeature/sdk/EventProvider.java

```diff
@@ -14,8 +18,15 @@
  *
  * @see FeatureProvider
  */
+@Slf4j
 public abstract class EventProvider implements FeatureProvider {
     private EventProviderListener eventProviderListener;
+    private static final int SHUTDOWN_TIMEOUT_SECONDS = 3;
```

Comment:
There's a `SHUTDOWN_TIMEOUT_SECONDS` var already in the `EventSupport` class. Maybe you can use that?

Reply:
That is where I got it from, but didn't want to couple the 2 classes together in this case, can do but since they are different Thread Pools, executing different kind of tasks, it feels like coincidental duplication but not actually the "same thing"

Reply:
You're right that it's not really the "same thing", but I'm not sure there's much value in there values ever deviating from each-other. :thinking: